doc/how-to/app/app_roles.rst

```diff
@@ -0,0 +1,476 @@
+.. _application_roles:
+
+Application roles
+=================
+
+An application role is a Lua module that implements specific functions or logic.
+You can enable or turn off a particular role for certain instances in a :ref:`configuration <configuration>` without restarting these instances.
+A role is run when a configuration is loaded or reloaded.
+
+Roles can be divided into the following groups:
+
+-   Roles provided by Tarantool.
```

Comment:
```suggestion
-   Tarantool's built-in roles.
```
Nit: this sounds more natural to me (if this wording is correct in our terminology).